File: Seminar5/HomeWork/View/Menu.java
package HomeWork.View;

import java.util.Scanner;

public class Menu {
    private final Scanner scanner;

    public Menu(Scanner scanner) {
        this.scanner = scanner;
    }

    public int selectFunction() {
        System.out.print("""
                --------------------------------------
                Выберите действие:
                --------------------------------------
                1. сложение
                2. вычитание
                3. умножение
                4. деление
                --------------------------------------
                        """);
        return scanner.nextInt();
    }

    public int selectCalc() {
        System.out.print("""
                --------------------------------------
                Выберите тип чисел:
                --------------------------------------
                1. рациональные
                2. комплексные
                --------------------------------------
                        """);
        return scanner.nextInt();
    }

}

File: FinalTaskCalculator/View/Menu.cs
using System;

namespace View
{
    public class Menu
    {
        public int SelectFunction()
        {
            Console.WriteLine(
                    "--------------------------------------\n" +
                    "Выберите действие:\n" +
                    "--------------------------------------\n" +
                    "1. сложение\n" +
                    "2. вычитание\n" +
                    "3. умножение\n" +
                    "4. деление\n" +
                    "--------------------------------------\n");
            return Convert.ToInt32(Console.ReadLine());
        }

        public int SelectCalc()
        {
            Console.WriteLine(
                    "--------------------------------------\n" +
                    "Выберите тип чисел:\n" +
                    "--------------------------------------\n" +
                    "1. рациональные\n" +
                    "2. комплексные\n" +
                    "--------------------------------------\n"
                            );
            return Convert.ToInt32(Console.ReadLine());
        }
    }
}